packages/react-jsx-highcharts/src/utils/warnings.js

```diff
@@ -51,6 +53,7 @@ const moduleToVarName = {
 };
 
 const moduleToSeriesTypes = {
+  annotations: ['annotations'],
```

Comment:
Maybe worth renaming this var to `moduleToFeatureMap` ?

Reply:
I couldn't figure good name for it, so thank you!